lib/Service/RuleService.php

```diff
@@ -154,7 +176,41 @@ private function processSoftwareCatalogusRule(Rule $rule, array $data): array
         }
         
         // foreach ($voorzieningGebruiken as $voorzieningGebruik) {
-        //     $data['body']['views'][] = $voorzieningGebruik;
+        //     $voorzieningGebruik = $voorzieningGebruik->jsonSerialize();
+        // }
+
+        // // Add organisaties (leveranciers) to response data
+        // foreach ($organisaties as $organisatie) {
+        //     $organisatie = $organisatie->jsonSerialize();
+
+        //     $newUuid = Uuid::v4();
+        //     $elementId = "id-{$newUuid}";
+        //     $data['body']['elements'][] = [
+        //         'identifier' => $elementId,
+        //         'name' => $organisatie['naam'],
+        //         'name-lang' => 'nl',
+        //         'documentation' => $organisatie['beschrijving'],
+        //         'documentation-lang' => 'nl',
+        //         'type' => 'BusinessActor',
+        //         'properties' => [
+        //             0 => [
+        //                 'propertyDefinitionRef' => 'id-c3355444b6cb8fb34b62e241dd073043', // SWC type
+        //                 'value' => 'Leverancier',
+        //             ],
+        //             1 => [
+        //                 'propertyDefinitionRef' => 'propid-2', // Object ID
+        //                 'value' => $newUuid,
+        //             ],
+        //             2 => [
+        //                 'propertyDefinitionRef' => 'propid-39', // URL
+        //                 'value' => '',
+        //             ],
+        //         ],
+        //     ];
+        // }
+
+        // foreach ($voorzieningAanbod as $voorzieningAanbod) {
+        //     $voorzieningAanbod = $voorzieningAanbod->jsonSerialize();
         // }
```

Comment:
Is there a reason for this long commented piece of code?

Reply:
This code has not been tested enough, it is out of scope for now, but we will need it later on. For now commented out because we quickly want to merge a fix that will make sure the Exports/downloads are valid to be imported into Archi